docker/README.md

```diff
@@ -1,13 +1,8 @@
 Sandbox Docker
 ==============
 
-- Uses `opendatacube/geobase` images as base
 - Username is `jovyan` to match "typical python notebook dockers"
 - `/env/..` contains python environment, writable by `jovyan` user
-- Two stage build
```

Comment:
This makes the image size smaller

Reply:
Agreed a two-stage build is the preferred final goal, but it can also add unwanted complexity when getting things working after major changes.  I'd be happy with a single-stage build for initial testing, and split the build in another PR after everything has been tested and is working.